doc/nrf/libraries/bluetooth/services/fast_pair/adv_manager.rst

```diff
@@ -0,0 +1,241 @@
+* **Payload synchronization**: Ensures advertising payload updates are synchronized with RPA rotations.
+* **Extension compatibility**: Is compatible with the Fast Pair extensions that are supported by the :ref:`bt_fast_pair_readme` module (for example, the Find My Device Network extension).
+
+  .. include:: /includes/fast_pair_fmdn_rename.txt
+
+* **Use case support**: Provides tailored behavior for specific Fast Pair device types (for example, locator tags).
+
+.. note::
+   The module is designed to work in cooperative thread context.
+
+.. _bt_fast_pair_adv_manager_config:
+
+Configuration
+*************
+
+The Fast Pair Advertising Manager is enabled with the :kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER` Kconfig option.
+As this is a helper module, the Kconfig option is only configurable if the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option is enabled (see the :ref:`bt_fast_pair_readme` documentation for more details).
+
+With the :kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER` Kconfig option enabled, the following Kconfig options are available for this module:
+
+* :kconfig:option:`BT_FAST_PAIR_ADV_MANAGER_USE_CASE` - This Kconfig choice is defined to select the Fast Pair use case (device type) and to enable additional advertising behaviours that are specific to the chosen use case.
+
+  * :kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER_USE_CASE_UNSPECIFIED` - The option provides a neutral configuration without use case specific behaviors.
+    By default, the :kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER_USE_CASE_UNSPECIFIED` option is selected.
+  * :kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER_USE_CASE_LOCATOR_TAG` - The option provides a tailored configuration for Fast Pair locator tags that use the FMDN extension.
+    It depends on the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN` Kconfig option.
+
+    * :kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER_USE_CASE_LOCATOR_TAG_CLOCK_SYNC_TRIGGER` - The option enables the clock synchronization trigger for locator tag devices.
+      The option is enabled by default.
+
+Dependencies
+============
+
+The :kconfig:option:`CONFIG_BT_FAST_PAIR_ADV_MANAGER` Kconfig option depends on several Bluetooth LE features:
+
+* :kconfig:option:`CONFIG_BT_PERIPHERAL` - Peripheral role support
+* :kconfig:option:`CONFIG_BT_EXT_ADV` - Extended advertising support
+* :kconfig:option:`CONFIG_BT_PRIVACY` - Bluetooth privacy support
+* :kconfig:option:`CONFIG_BT_SMP` - Security Manager Protocol support
+* :kconfig:option:`CONFIG_BT_ADV_PROV` - Advertising data provider support
+* :kconfig:option:`CONFIG_BT_ADV_PROV_FAST_PAIR` - Fast Pair advertising data provider support
+
+Implementation details
+**********************
+
+This module uses Zephyr Bluetooth subsystem APIs to manage the Fast Pair advertising set and the connections that are derived from it.
```

Comment:
Does the module allow for more advertising sets coexistance? How is it handled? What happens when someone connects to them?